### PR TITLE
Feat/add back to top button

### LINKED ISSUE #6: Add 'back to top' button
Since the page is quite long, I feel like a 'back to top' button would be useful.

### PR DESCRIPTION
**Pull request by bbenefield89**
fixes #6 

### Feature Name
Add Back To Top Button

### Category
UI/UX

### Info
```
This feature adds a small button on the bottom right of the browser that when clicked will take the user to the top of the page.
```

### Example
```js
const handleBackToTop = () => {
    window.scroll({ top: 0, behavior: 'smooth' });
  };
```

### CODE REVIEW

File: src/components/backtotop.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+fontawesome.library.add(faArrowUp);
+
+const BackToTop = () => {
+  const handleBackToTop = () => {
+    window.scroll({ top: 0, behavior: 'smooth' });
+  };
+  
+  return (
+    <div className="back-to-top" onClick={ () => handleBackToTop() } onScroll={ () => alert() } >
```
Comment (melanieseltzer) [inline src/components/backtotop.js]: What does `onScroll={() => alert()}` do? Do we need it?

Also super nitpicky format thing, can we organize the div like 

```
 <div
 	className...
 	onClick...
```

etc

**Comment by melanieseltzer** [comment]
This looks awesome! 👍

File: src/components/backtotop.js (hunk 1 above)
Comment (bbenefield89) [reply src/components/backtotop.js]: We definitely do not need this, must have left it in there :rofl: 

I'll remove it and the style changes in just a moment.

**Comment by bbenefield89** [comment]
Thanks. I've updated the PR, removed the `onScroll` and format the `div`.

**Comment by melanieseltzer** [comment]
Thank you 🎉

**Pull request merged**